rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/bulk/11_dynamic_templates.yml

```diff
@@ -139,11 +139,11 @@
           - { "location": "41.12,-71.34"} # ok as fields are defined
   - match: { errors: true }
   - match: { items.0.index.status: 400 }
-  - match: { items.0.index.error.type: mapper_parsing_exception }
-  - match: { items.0.index.error.reason: "Can't find dynamic template for dynamic template name [bar_foo] of field [foo_location]"}
+  - match: { items.0.index.error.type: document_parsing_exception }
+  - match: { items.0.index.error.reason: "[-1:-1] failed to parse: Can't find dynamic template for dynamic template name [bar_foo] of field [foo_location]"}
```

Comment:
That -1:-1 isn't great? Shall we omit printing out the location when it's not set? Or should it be set to the location where the field appears first ?

Reply:
I've changed this so that it doesn't output the location when it's `UNKNOWN`